### PR TITLE
Fixing advanced payout - Default to Bountied Amount for Payment, etc.

### LINKED ISSUE #2294: Advanced Pay Fix - Default to Bountied Amount for Payment, Add Guidance on Overage
### User Story

As a funder I'd like the advanced pay feature to be more intuitive and provide guidance on where funds are coming from.

### Description

Update the build so that:
- All funds come from the bountied amount.
- When overage is detected, an alert message notifies the user of the difference and where the funds will be paid from. This is not meant to be an error, but more of a warning.
- Clean up the Payout preview (remove confusing terms and unecessary information).

### Definition of Done

- [ ] Remove Return Bounty Stake Checkbox
- [ ] Default all payments to pay from the Initial Funded Bounty Amount

**Alert**
- [ ] Add an alert (bg color #E66700, #FFF for copy) when user exceeds 100% or Bountied value in ETH.
- [ ] Add wallet information and overage amount to the alert.

**Payout Preview**
- [ ] Display the overage amount on the payout preview in #E66700
- [ ] Remove refunded Eth
- [ ] Remove mention of bounty stake

**Other**
- [ ] If multiple users have been added to the pay list, and a user is removed, the payout preview should reflect the removal.

### Related Issues

This ticket is part of a series of fixes for Advanced Pay
- **[Issue 2215](https://github.com/gitcoinco/web/issues/2215)** Display Tooltip **Where is My Money Going** Bountied to XXX
- Default to Bountied Amount for Payment
- **[Issue 2240](https://github.com/gitcoinco/web/issues/2240)** Add Auto Complete & Make it clearer who I'm paying (verified/unverified)

### Design 
screen shot 2018-09-28 at 9 32 36 am

### PR DESCRIPTION
**Pull request by Anish-Agnihotri**
##### Description

This PR resolves #2294 by introducing: 

1. Alerts upon advanced payment > 100%.
2. Automatic table altering upon X click. 
3. Other misc. payout preview cleaning (removing mentions of bounty stake, etc.).

[See it in action!](https://streamable.com/dirw9)

##### Checklist

- [x] Remove Return Bounty Stake Checkbox
- [x] Default all payments to pay from the Initial Funded Bounty Amount
- [x] Add an alert (bg color #E66700, #FFF for copy) when user exceeds 100% or Bountied value in ETH.
- [x] Add wallet information and overage amount to the alert.
- [x] Display the overage amount on the payout preview in #E66700
- [x] Remove refunded Eth
- [x] Remove mention of bounty stake
- [x] If multiple users have been added to the pay list, and a user is removed, the payout preview should reflect the removal.
- [x] This issue is tested

##### Affected core subsystem(s)

UI is the only core system affected, since all changes are aesthetic. 

##### Testing

Tested across two environments across two platforms (W10, Mojave). Feel free to test yourself as well.

##### Refers/Fixes

Fixes #2294.

### CODE REVIEW

**Comment by mbeacom** [comment]
Additionally, can you resolve the merge conflict?

**Comment by Anish-Agnihotri** [comment]
@mbeacom 

Thanks, working on resolving that and the linting errors right now.

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 1]
```diff
@@ -181,7 +223,36 @@ <h5>{% trans 'Payout Preview' %}</h5>
...
+        margin-bottom: 5px;
+        display: none;
+        padding-left: 15px;
+      }
+      .overageAlert > div:first-child {
+        width: 25px;
+        float: left;
+        padding-right: 10px;
+      }
+      .overageAlert> div:last-child {
```
Comment (pinkiebell) [inline app/dashboard/templates/bulk_payout_bounty.html]: ` > ` - cosmetic whitespace missing 😄

**Comment by pinkiebell** [comment]
LGTM; Just that one cosmetic one-liner to fix 😄

**Comment by Anish-Agnihotri** [comment]
> LGTM; Just that one cosmetic one-liner to fix 😄

Thanks for the review! Just pushed out the fix in the new commit :).

File: app/assets/v2/js/pages/bulk_payout.js [hunk 2]
```diff
@@ -166,7 +170,16 @@ $(document).ready(function($) {
       _alert('You do not have any transactions to payout.  Please add payees to the form.', 'error');
       return;
     }
+    var usernames = $('.username');
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bulk_payout.js]: encourage moving to let / const as opposed to var ? 
cause we've got the ES6 support coming in this week

File: app/assets/v2/js/pages/bulk_payout.js [hunk 3]
```diff
@@ -166,7 +170,16 @@ $(document).ready(function($) {
...
+    
+    for (var i = 0; i < usernames.length; i++) {
+      var username = usernames[i].textContent.trim();
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/bulk_payout.js]: [same](https://github.com/gitcoinco/web/pull/2365/files#r223200671)

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 4]
```diff
@@ -120,6 +153,15 @@ <h5>{% trans 'Payout Preview' %}</h5>
                     </a>
                   </div>
                 </div>
+
```
Comment (thelostone-mc) [inline app/dashboard/templates/bulk_payout_bounty.html]: remove the extra line ?

**Comment by thelostone-mc** [comment]
Hmm I see changes from the autcomplete and tooltip PR repeating here. Any idea why that's happening ? 🤔 [here](https://github.com/gitcoinco/web/pull/2365/files#diff-c7729908a886c5d5dc2d91ff4c235eaeR23), [here](https://github.com/gitcoinco/web/pull/2365/files#diff-c7729908a886c5d5dc2d91ff4c235eaeR275) and a few other places.

Could you double check to make sure only your changes are preset in the PR ?

**Comment by Anish-Agnihotri** [comment]
@thelostone-mc I had to add those since there seemed to be a merge error otherwise. I can rebase from default, restructure, and push out another commit tonight. 

And, sure, I'll change variable declaration to follow ES6 standards as well.

**Pull request closed without merging**